conftest.py

```diff
@@ -221,6 +222,24 @@ def pytest_terminal_summary(
         terminalreporter.write(" ".join(["pytest"] + ids))
         terminalreporter.write("\n" * 2)
 
+        # If some tests failed at installing mlflow, we suggest using `--serve-wheel` flag.
+        # Some test cases try to install mlflow via pip e.g. model loading. They pins
+        # mlflow version to install based on local environment i.e. dev version ahead of
+        # the latest release, hence it's not found on PyPI. `--serve-wheel` flag was
+        # introduced to resolve this issue, which starts local PyPI server and serve
+        # an mlflow wheel based on local source code.
+        # Ref: https://github.com/mlflow/mlflow/pull/10247
+        msg = f"No matching distribution found for mlflow=={VERSION}"
+        for rep in failed_test_reports:
+            for text in (rep.longreprtext, rep.capstdout, rep.capstderr):
+                if msg in text:
+                    terminalreporter.section("HINTS", yellow=True)
+                    terminalreporter.write(
+                        f"\033[93mFound test(s) that failed with {msg!r}. Adding"
+                        " --serve-wheel` flag to your pytest command may help.\n\n\033[0m"
+                    )
+                    break
```

Comment:
```suggestion
                    terminalreporter.write(
                        f"\033[93mFound test(s) that failed with {msg!r}. Adding"
                        " --serve-wheel` flag to your pytest command may help.\n\n\033[0m",
                        yellow=True,
                    )
                    break
```

Can we use `yellow=True`?

Reply:
oh shame, didn't notice that is used one line ahead